tests/onnxruntime/test_modeling.py

```diff
@@ -817,18 +934,47 @@ def test_compare_to_io_binding(self, model_arch):
         gc.collect()
 
 
-class ORTModelForSequenceClassificationIntegrationTest(unittest.TestCase):
-    SUPPORTED_ARCHITECTURES = (
-        "distilbert",
+class ORTModelForSequenceClassificationIntegrationTest(ORTModelTestMixin):
+    SUPPORTED_ARCHITECTURES = [
+        "albert",
+        "bart",
         "bert",
+        "big_bird",
+        "bigbird_pegasus",
+        "bloom",
         "camembert",
-        "roberta",
-        "xlm-roberta",
+        "convbert",
+        "data2vec_text",
+        "deberta",
+        "deberta_v2",
+        "distilbert",
         "electra",
-        "albert",
-        "bart",
+        "flaubert",
+        "gpt2",
+        "gpt_neo",
+        "gptj",
+        "ibert",
+        # TODO: these two should be supported, but require image inputs not supported in ORTModel
+        # "layoutlm"
+        # "layoutlmv3",
         "mbart",
-    )
+        "mobilebert",
+        # "perceiver",
+        "roberta",
+        "roformer",
+        "squeezebert",
+        "xlm",
+        "xlm_roberta",
+    ]
+
+    ARCH_MODEL_MAP = {
```

Comment:
What's the use of `ARCH_MODEL_MAP`?

Reply:
Some architectures like `perceiver` can't have the same model used for different tasks (e.g. sequence classification and image classification). It is similar to https://github.com/huggingface/optimum/blob/6e1194133cbf25be6d55c6e77f5b9b133a43ec75/tests/exporters/exporters_utils.py#L74-L76

Currently this is not used because perceiver does not work with ORTModel but it can be useful in the future.